packages/material-ui/src/Paper/Paper.js

```diff
@@ -42,15 +45,27 @@ const Paper = React.forwardRef(function Paper(props, ref) {
     ...other
   } = props;
 
+  const themeVariantsClasses = useThemeVariants(
+    {
+      ...props,
+      component: Component,
+      square,
+      elevation,
+      variant,
+    },
+    'MuiPaper',
+  );
+
   return (
     <Component
       className={clsx(
         classes.root,
+        classes[variant],
         {
           [classes.rounded]: !square,
           [classes[`elevation${elevation}`]]: variant === 'elevation',
```

Comment:
```suggestion
          [classes[`elevation${elevation}`]]: variant !== 'outlined',
```
Should we rather change this now? As new variants may not necessarily want to disabled the elevation

Reply:
Why not, the last time I had to customize the Paper, I had removed the elevation. People should be careful when using elevation, it can easily go wrong (look horrible).

Reply:
Ok will leave it as it is